python-lib/recipe_config_loading.py

```diff
@@ -0,0 +1,33 @@
+import logging
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO, format='timeseries-preparation plugin %(levelname)s - %(message)s')
```

Comment:
You can remove this line as it does not do anything.

Reply:
ha ok, thanks!